packages/core/admin/server/src/validation/project-settings.ts

```diff
@@ -1,45 +1,45 @@
-import { yup, validateYupSchemaSync } from '@strapi/utils';
+import { z } from 'zod';
+import { validateZod } from '@strapi/utils';
 
 const MAX_IMAGE_WIDTH = 750;
 const MAX_IMAGE_HEIGHT = MAX_IMAGE_WIDTH;
 const MAX_IMAGE_FILE_SIZE = 1024 * 1024; // 1Mo
-const ALLOWED_IMAGE_FILE_TYPES = ['image/jpeg', 'image/png', 'image/svg+xml'];
 
-const updateProjectSettings = yup
+const updateProjectSettings = z
```

Comment:
should we look at trying to move some of these schemas into the shared folder so we can use them on the FE as well as the BE?

Reply:
very excited to see this!